www-servers/lighttpd/lighttpd-1.4.39.recipe

```diff
@@ -110,41 +75,155 @@ BUILD_REQUIRES="
+			s|\"/var\(/log/lighttpd\"\)|\"$localStateDir\1|" \
+		-e "/^var\.server_root/ \
+			s|\"/srv/www\"|\"$dataDir/lighttpd/www\"|" \
+		-e "/^var\.server_root/ \
+		  a #var.server_root = \"/$relativeSharedStateDir/www\"" \
+		-e "/^var\.state_dir/ s|\"/var/run\"|\"$localStateDir\"|" \
+		-e "/^var\.home_dir/ \
+		  s|\"/var/lib/lighttpd\"|\"$localStateDir/lib/lighttpd\"|" \
+		-e "/^var\.conf_dir/ \
+		  s|\"/etc\(/lighttpd\"\)|\"/system/$relativeSettingsDir\1|" \
+		-e "/^var\.cache_dir/ \
+			s|\"/var\(/cache/lighttpd\"\)|\"$localStateDir\1|" \
+		-e "/^server\.use-ipv6 / s|\"enable\"|\"disable\"|" \
+		-e "s|^server\.username |#&|" \
+		-e "s|^server\.groupname |#&|" \
+		-e "s|^server\.max-fds |#&|" \
+		-e "s|^server\.max-connections |#&|" \
+		-e "s|^server\.network-backend |#&|" \
+		-e "/^server\.event-handler/ \
+			s|\"linux-sysepoll\"|\"select\"|" \
+		-e "/^server\.upload-dirs/ \
+			s|\"/var/tmp\"|\"$localStateDir/tmp\"|" \
+		-e "/^#ssl\.pemfile/ \
+			s|\"/etc/ssl\(/private/lighttpd.pem\"\)|\"$dataDir\1|" \
+		-e "/^include \"modules\.conf\"/ \
+			a index-file.names += ( \"index.lighttpd.html\" )" \
+		-i $settingsDir/lighttpd/lighttpd.conf
+
+	cp doc/config/modules.conf $settingsDir/lighttpd
+	sed \
+		-e "s|^#\(include \"conf\.d/ssi\.conf\"\)|\1|" \
+		-i $settingsDir/lighttpd/modules.conf
+
+	cp doc/config/conf.d/*.conf $settingsDir/lighttpd/conf.d
+	sed \
+		-e "/^ssi\.extension/ \
+			a index-file.names += ( \"index.shtml\" )" \
+		-i $settingsDir/lighttpd/conf.d/ssi.conf
+
+	mkdir -p $dataDir/lighttpd/www/htdocs
+
+# Create and install a static index.lighttpd.html test page from the template:
+
+	cp	$portDir/additional-files/index.template \
+		$dataDir/lighttpd/www/htdocs/index.lighttpd.html
+	sed \
+		-e "s|@SYSTEM@|`uname -a`|" \
+		-e "s|@GETARCH@|`getarch`|" \
+		-e "s|@HOSTNAME@|`uname -n`|" \
+		-e "s|@LIGHTTPD_VERSION@|lighttpd/$portVersion|" \
+		-i $dataDir/lighttpd/www/htdocs/index.lighttpd.html
 
-	prepareInstalledDevelLibs mod_access mod_accesslog mod_alias mod_auth \
-		mod_cgi mod_cml mod_compress mod_dirlisting mod_evasive mod_evhost \
-		mod_expire mod_extforward mod_fastcgi mod_flv_streaming mod_indexfile \
-		mod_magnet mod_mysql_vhost mod_proxy mod_redirect mod_rewrite \
-		mod_rrdtool mod_scgi mod_secdownload mod_setenv mod_simple_vhost \
-		mod_ssi mod_staticfile mod_status mod_trigger_b4_dl mod_userdir \
-		mod_usertrack mod_webdav
+# Create and install the SSI index.shtml test page:
 
-	packageEntries devel $developDir
+	cp	$portDir/additional-files/index.template \
+		$dataDir/lighttpd/www/htdocs/index.shtml
+	sed \
+		-e "s|@SYSTEM@|<!--#exec cmd=\"/bin/uname -a\"-->|" \
+		-e "s|@GETARCH@|<!--#exec cmd=\"/bin/getarch\"-->|" \
+		-e "s|@HOSTNAME@|<!--#echo var=\"HTTP_HOST\"-->|" \
+		-e "s|@LIGHTTPD_VERSION@|<!--#echo var=\"SERVER_SOFTWARE\"-->|" \
+		-i $dataDir/lighttpd/www/htdocs/index.shtml
 }
 
 TEST()
 {
 	make check
+	lighttpd -t -f /system/settings/lighttpd/lighttpd.conf
```

Comment:
Done.
Not really usefull indeed, as the resulting config file can only change if the recipe changes.
The only advantage would be to check that the runtime has been correctly built, but since most recipes don't have a real **`TEST()`** (other than a **`make check`**) adding an extra command here is not necessary.